connector/spanmetricsconnector/README.md

```diff
@@ -163,7 +164,6 @@ connectors:
     exemplars:
       enabled: true
     exclude_dimensions: ['status.code']
-    dimensions_cache_size: 1000
     aggregation_temporality: "AGGREGATION_TEMPORALITY_CUMULATIVE"    
```

Comment:
Why was this removed?

Reply:
https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/83a13e6ff3eb95e9793cb9040aa3b64612395888/connector/spanmetricsconnector/README.md?plain=1#L113C4-L113C25

This setting has been deprecated